src/shogun/neuralnets/NeuralLayers.h

```diff
@@ -47,6 +47,7 @@ namespace shogun
  * Uses builder-style syntax to describe layers
  * one by one.
  *
+ * While adding a new layer, also add it's prototype in SWIG with %newobject.
```

Comment:
This both contains a grammar error "it's -> its", and also doesnt make any sense to me :)

rather remove, or replace with something meaningful. Recall that users of Shogun dont know what SWIG is. Rather say that every methods returns a reference to the same object or so